tools/team_composer.py

```
@@ -0,0 +1,941 @@
+                char.vuln_effects.add(eid)
+            # 增益
+            if eid.startswith("BUFF_"):
+                char.buff_effects.add(eid)
+            # 减益
+            if eid.startswith("DEBUFF_"):
+                char.debuff_effects.add(eid)
+            # 异常状态
+            if eid.startswith("STATUS_") and eid not in (
+                "STATUS_SPELL_INFLICT", "STATUS_SPELL_BURST",
+                "STATUS_SPELL_ANOMALY", "STATUS_BROKEN",
+                "STATUS_STAGGER",  # 失衡：基础异常，不算协同
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)